prowler/config/config.yaml

```diff
@@ -316,6 +316,24 @@ aws:
 
     ]
 
+  #AWS S3 Configuration
+  # Recommended LifeCycle expiration days
+  min_lifecycle_expiration_days: 1
+  max_lifecycle_expiration_days: 36500
+  # Recommended LifeCycle transition days
+  min_lifecycle_transition_days: 1
+  max_lifecycle_transition_days: 36500
+  # Recommended LifeCycle transition storage classes
+  lifecycle_transition_storage_classes:
+    [
+      "STANDARD_IA",
+      "INTELLIGENT_TIERING",
+      "ONEZONE_IA",
+      "GLACIER",
+      "GLACIER_IR",
+      "DEEP_ARCHIVE",
+    ]
+
```

Comment:
I would not use configurable values, only check if the bucket has a lifecycle enabled since this only would make sense if we make the parameters optional and only 3, not 5.